diff --git a/spec/requests/user_pages_spec.rb b/spec/requests/user_pages_spec.rb
index 6c238bc..ffec682 100644
--- a/spec/requests/user_pages_spec.rb
+++ b/spec/requests/user_pages_spec.rb
@@ -47,6 +47,14 @@ describe "UserPages" do
 			it "should create a user" do
 				expect { click_button submit }.to change(User, :count).by(1)
 			end
+
+			describe "after saving the user" do
+				before { click_button submit }
+				let(:user) { User.find_by(email: 'user@example.com') }
+
+				it { should have_title(user.name) }
+				it { should have_selector('div.alert.alert-success', text: 'Welcome') }
+			end
 		end
 	end
 end
\ No newline at end of file
